flake.nix

```diff
@@ -4,25 +4,30 @@
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
     flake-utils.url = "github:numtide/flake-utils";
+    pnpm2nix.url = "github:nzbr/pnpm2nix-nzbr";
     drpc.url = "github:storj/drpc/v0.0.33";
   };
 
-  outputs = { self, nixpkgs, flake-utils, drpc }:
+  outputs = { self, nixpkgs, flake-utils, drpc, pnpm2nix }:
     flake-utils.lib.eachDefaultSystem (system:
       let
         # Workaround for: terraform has an unfree license (‘bsl11’), refusing to evaluate.
         pkgs = import nixpkgs { inherit system; config.allowUnfree = true; };
         formatter = pkgs.nixpkgs-fmt;
         nodejs = pkgs.nodejs-18_x;
-        yarn = pkgs.yarn.override { inherit nodejs; };
```

Comment:
We might want to either remove `yarn` from the packages below (since otherwise it could use the wrong Node version) or put this line back.  Maybe remove makes more sense since `pnpm` should be used anyway!

Reply:
Yeah, remove